services/conflict/groupMediation.ts

```
@@ -0,0 +1,666 @@
+        }
+        if (completionScore < 80) {
+          recommendations.push('Ensure all participants have spoken');
+          recommendations.push('Confirm understanding of ground rules');
+        }
+        break;
+
+      case 'perspective_sharing':
+        // Check if all participants have shared their perspective
+        const sharedPerspectives = conflict.participants.filter(p => 
+          p.perspective.summary && p.perspective.summary.length > 0
+        ).length;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)